packaging/cloudify-agents.spec

```diff
@@ -7,6 +7,12 @@
 %define _source_payload w0.gzdio
 %define _binary_payload w0.gzdio
 
+# this prevents networkx<2 failure in RH8
```

Comment:
RH8 build by default mangles shebangs, which fails with the test files of networkx<2 due to RH8 having python3 rather than python as its python command (e.g. https://github.com/networkx/networkx/tree/v1.11/networkx/tests). 
Since we can't change files of an external library, we just prevent this mangling in the first place